spec/requests/auctions_spec.rb

```diff
@@ -0,0 +1,334 @@
+require 'rails_helper'
+
+RSpec.describe AuctionsController do
```

Comment:
I think this file was accidentally re-added (split into two request specs in a recent PR, this one should be deleted)